commands/acl-dryrun.md

```diff
@@ -0,0 +1,18 @@
+Simulate the execution of a given command by a given user.
+This command can be used to test the permissions of a given user without having to enable the user or cause the side effects of running the command.
+
+@return
+
+@simple-string-reply: `OK` on success.
+@bulk-string-reply: An error describing why the user can't execute the command.
+
+@examples
+
+```
+> ACL SETUSER FOO +SET ~*
+"OK"
+> ACL DRYRUN USER SET FOO BAR
+"OK"
+> ACL DRYRUN USER GET FOO BAR
+"This user has no permissions to run the 'GET' command"
```

Comment:
This example is confusing. Is FOO the username or the key or both? Is USER the username or a subcommand?

Better use some username like "virginia" that you used somewhere else.